tests/Unit/TestHelpers.hpp

```diff
@@ -233,6 +233,9 @@ void test_serialization(const T& t) {
 /// \brief Test the serialization of a derived class via a base class pointer
 /// \example
 /// \snippet Test_PupStlCpp11.cpp example_serialize_derived
+/// \tparam B the base class
+/// \tparam D the derived class
+/// \tparam Args arguments passed to a constructor of the derived class
```

Comment:
Singe `Args` is deduced I had in mind documenting `\param args` instead (or both). Sorry for not being clear on this.